provider/src/main/java/com/synopsys/integration/alert/provider/blackduck/BlackDuckProviderDataAccessor.java

```diff
@@ -314,4 +319,12 @@ private int computeTotalCount(BlackDuckPageResponse<?> blackDuckPageResponse, in
         return blackDuckResponsesTransformer.getSomeMatchingResponses(pagedRequest, responseClass, searchFilter, pagedRequest.getLimit());
     }
 
+    private String truncateDescription(@Nullable String originalDescription) {
+        String trimmedDescription = StringUtils.trimToEmpty(originalDescription);
+        if (trimmedDescription.length() > PROJECT_DESCRIPTION_MAX_CHARS) {
+            return StringUtils.truncate(trimmedDescription, PROJECT_DESCRIPTION_MAX_CHARS) + ". . .";
```

Comment:
Would it be beneficial to truncate to a few under 256 then add the new content? Not sure the most efficient way of storing data in our current DB.

Reply:
We're really just trying to minimize UI bloat here whereas before we were writing to the DB, so it needed to be more precise.